Core/Utilities/TimeLeft/LSFTimeLeft.py

```diff
@@ -77,9 +69,9 @@ def __init__( self ):
       cmd = '%s/lshosts -w %s' % ( self.bin, self.cpuRef )
       result = runCommand( cmd )
       if result['OK']:
-        # At CERN this command will return an error since there is no host defined 
+        # At CERN this command will return an error since there is no host defined
         # with the name of the reference Host.
-        lines = result['Value'].split( '\n' )
+        lines = str( result['Value'] ).split( '\n' )
```

Comment:
You are right, but pylint was confused by these lines and was creating a fatal error, saying 
ID:no-member Instance of 'tuple' has no 'split' member
As I could not find out how to fix it, I forced it to be a string...